.travis.yml

```diff
@@ -5,6 +5,16 @@ php:
   - 5.6
   - 7.2
 
+env:
+  - PRESTASHOP_VERSION=latest
+
+matrix:
+  include:
+    - php: 7.2
+      env: PRESTASHOP_VERSION=1.7.3.0
+    - php: 7.2
+      env: PRESTASHOP_VERSION=1.7.1.2
```

Comment:
can we add every one of theme like 1.7.0.0 and 1.7.2.0 ?

Reply:
We balance between tests, and time needed to run them.

Our objective is to check the backward compatibility with the biggest range of PrestaShop versions. So, checking with the first and the last PrestaShop 1.7.x.x is enough. All other versions in this range have been checked by the core team.

Reply:
hum agree but then we must add 1.7.0.0 because this is where most of the changes happened no ?

Reply:
but now we only have 1.7.0.0 shouldn't we have the latest one too ?

Reply:
We do have both:

![image](https://user-images.githubusercontent.com/6768917/70427753-3a4b6b00-1a6d-11ea-8099-9af85908d99a.png)

Travis configuration can be confusing sometimes. We already have the 2 builds with the default PS version set as `latest`, on which I added a specific build for PHP 7.2 and PS 1.7.0.0.

Reply:
ok i see seems good then !